### ISSUE TITLE
Suppress rest buttons in character sheets

### ISSUE THREAD

**Issue by ludifu** [fantasycalendar/FoundryVTT-RestRecovery#136]
If players inadvertently press rest buttons from their character sheets, when they should not because they are not entitled to rest benefits, this cannot easily be undone (hp, spell slots, etc.). Also triggering a rest for the remaining players (for advancing time, exhaustion integration, reset of per day abilities etc.) is then not possible anymore, because players having used their own buttons would then benefit from 2 rests.

I would like to have a toggle switch in the settings to enable / disable rest buttons for players.

Currently I need to remind players to not use the default buttons. This is not reliable though.

**Comment by Haxxer** [comment]
Could you export your current setup through the settings menu (in the top bar)? Clicking on the rest buttons should not immediately finish the rest, at least not in the default sheet or Tidy5e.

**Comment by ludifu** [comment]
It seems what I wrote provoked misunderstanding.

I was referring to the standard rest buttons in the character sheet, not anything from the module. The module is working perfectly fine for me.

However in the meantime I learned that disabling the rest buttons in the standard character sheet is possible using Illandril's Character Sheet Lockdown (5e) module. So while my request may be a suitable feature for RestRecovery as well, there is another way to do this. I would still like this in RestRecovery but the appropriate priority is probably very low. Unless you want to keep this request open yourself I suggest to close this.
